Workbooks/WorkbooksMetadata.json

```diff
@@ -970,5 +970,18 @@
   "templateRelativePath": "MITREAttack.json",
   "subtitle": "",
   "provider": "Azure Sentinel community"
+  },
+  {
+  "workbookKey": "InvestigationInsightsWorkbook",
+  "logoFileName": "Microsoft_logo.svg",
+  "description": "Help analysts gain insight into Incident and Entity data through the Investigation Insights Workbook.  This workbook provides common queries and detailed visualizations to help an analyst investigate suspicious activities quickly with an easy to use interface. Analysts can start their investigation from a Sentinel Incident, or by simply entering the entity data into the workbook manually.",
```

Comment:
please change it to the following:
Help analysts gain insight into incident and entity data through the Investigation Insights Workbook.  This workbook provides common queries and detailed visualizations to help an analyst investigate suspicious activities quickly with an easy to use interface. Analysts can start their investigation from a Sentinel incident, or by simply entering the entity data into the workbook manually.

Reply:
updated